packages/shared/src/contexts/payment/index.tsx

```diff
@@ -1,28 +1,38 @@
 import React from 'react';
 import type { ReactElement } from 'react';
 import dynamic from 'next/dynamic';
-import { isIOSNative } from '../../lib/func';
+import { checkIsExtension } from '../../lib/func';
+import type { PaddleSubProviderProps } from './Paddle';
 import { PaddleSubProvider } from './Paddle';
+import { ChromeExtensionProvider } from './ChromeExtension';
 import type { PaymentContextProviderProps } from './context';
+import type { StoreKitSubProviderProps } from './StoreKit';
+import { iOSSupportsPlusPurchase } from '../../lib/ios';
 
 const StoreKitSubProvider = dynamic(() =>
   import('./StoreKit').then((mod) => mod.StoreKitSubProvider),
 );
 
 export const PaymentContextProvider = ({
   children,
-  successCallback,
+  ...props
 }: PaymentContextProviderProps): ReactElement => {
-  if (isIOSNative()) {
```

Comment:
@omBratteng I've changed the function being used to check on whether to render the storekit provider. In-app browser for some reason turns true on with `isIOSNative`. But then we purchase through Paddle in that instance. Wdyt? Should this be safe or I should just check whether it is in-app (similar to how we decide if we redirect auth) along with the original function.

Reply:
Yeah I noticed this as well, not sure if `isIOSNative` worked before.